### ISSUE TITLE
VM install expects /etc/hosts entry on vm

### ISSUE THREAD

**Issue by reedstrm** [Connexions/cnx-deploy#73]
## Summary

When doing a fresh install using the vm environment, the install expects 'zope.local.cnx.org' in the hosts file on the VM, while the README instructions only say to put it in the local machine.

## Environment

This is a local vm deploy on ubuntu 16.04.1 (deploying to 16.04.1 as well)

**Comment by karenc** [comment]
I don't know if this is a good way of fixing it, getting the IP from the local machine: https://github.com/Connexions/cnx-deploy/pull/71/commits/06b010e37173ad4b7bb872010c497a28ccd09223

**Comment by DarkPrinceFrost** [comment]
Or we can add the host via ansible:

```
lineinfile: >
        dest=/etc/hosts
        line="127.0.0.1 zope.local.cnx.org"
        create=yes
```

**Comment by karenc** [comment]
`zope.local.cnx.org` is not `127.0.0.1`, we can possibly get it from `/sbin/ifconfig`.

**Comment by DarkPrinceFrost** [comment]
We can't have it set as a variable?

**Comment by pumazi** [comment]
Is this potentially addressed in #38?

**Comment by karenc** [comment]
Yes, I think this is a duplicate.

**Comment by karenc** [comment]
Fixed in #78
